src/main/java/oneus/GSMATCH/global/level/Level.java

```diff
@@ -0,0 +1,37 @@
+package oneus.GSMATCH.global.level;
+
+import lombok.RequiredArgsConstructor;
+import oneus.GSMATCH.domain.user.entity.UserEntity;
+import oneus.GSMATCH.domain.user.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class Level {
+    private final UserRepository userRepository;
+
+    void levelUp(Long id) {
+        UserEntity user = userRepository.findById(id).orElseThrow();
```

Comment:
```suggestion
        UserEntity user = userRepository.findById(id).orElseThrow(() -> new CustomException(NOT_MATCH_INFORMATION));
```

"사용자를 찾을 수 없음" 에러를 던져주세요!